Test/Classes/TestTextUtils.cs

```diff
@@ -79,7 +463,19 @@ public void StripDateFromNameSimilarDate()
         public void StripDateFromNameIgnoreDifferent()
         {
             DateTime date = new DateTime(2009, 02, 13);
-            Assert.Equal("Test 26/05/2010", TextUtils.StripDateFromName("Test 26/05/2010", date));
+
+            try
+            {
+                Assert.Equal("Test 26/05/2010", TextUtils.StripDateFromName("Test 26/05/2010", date));
+                Assert.Equal("Test 26/05/2011", TextUtils.StripDateFromName("Test 26/05/2011", date));
+                Assert.Equal("Test 2010/05/26", TextUtils.StripDateFromName("Test 2010/05/26", date));
+                Assert.Equal("Test 26/08/2010", TextUtils.StripDateFromName("Test 26/08/2010", date));
+                Assert.Equal("Test 06/05/2012", TextUtils.StripDateFromName("Test 06/05/2012", date));
+            }
+            catch
+            {
+                Assert.False(true);
+            }
```

Comment:
Again, this `try {} catch {}` should be removed.

Reply:
Thanks will work through the small issues; many try...catch to narrow down which of the tests were failing etc. have now removed those from the TestTextUtils.cs file.  Other general comments i'll progressively clean up (or remove) once i'm more sure that the code is doing all that we need for this issue.

cc59f18 was my doing; as part of testing this i was manually subscribing to and downloading 20 or 30 podcasts from a single provider and during this i found that the application would throw an 'Out of Memory' error; so i put in some temp code to work out why and recover those podcast files; once i worked out if i subscribe to 10 or less (manually) the error didnt occur. So i have since removed the code, so i'm thinking that despite all of this the database.cs file should appear unchanged.

I did notice the testing failed when i did the PR; and equally unsure why. I'm using Visual Studio on windows (sorry if already mentioned that) and i can manually 'Run All' tests and they show as passing in the IDE.  Am curious whether this problem goes away when i update the PR with a version of the TestTextUtil.cs without the try...catch entries. Any guidance on what else i should be looking for re that appreciated.

Reply:
> I did notice the testing failed when i did the PR; and equally unsure why. I'm using Visual Studio on windows (sorry if already mentioned that) and i can manually 'Run All' tests and they show as passing in the IDE. Am curious whether this problem goes away when i update the PR with a version of the TestTextUtil.cs without the try...catch entries. Any guidance on what else i should be looking for re that appreciated.

I had a quick look at a couple of the failures and it looks like something is throwing an exception.  Can't tell what though because of the try {} catch {} blocks wrapping the tests.  If you remove them as part of your updates we should hopefully be able to see what is failing.